django/db/models/base.py

```diff
@@ -569,6 +569,10 @@ def _get_pk_val(self, meta=None):
         return getattr(self, meta.pk.attname)
 
     def _set_pk_val(self, value):
+        for parent_link in self._meta.parents.values():
+            if parent_link:
```

Comment:
Under which circumstances is this `False`?

Reply:
It's `None` when you use a proxy model (parent) to create a child, see [test_proxy_inheritance_supported](https://github.com/django/django/blob/4fe486520f9275d88f25a867706b974886ad1b54/tests/bulk_create/tests.py#L78-L82).